src/fdb5/database/Index.h

```diff
@@ -108,7 +108,9 @@ class IndexBase : public eckit::Counted {
     void decodeCurrent(eckit::Stream& s, const int version);
```

Comment:
several (above/below) redundant `const int`; can be removed in declaration (but keep them in source)